README.md

```diff
@@ -1,11 +1,33 @@
 # pythonparser
 [![elena-lyulina](https://circleci.com/gh/elena-lyulina/pythonparser/tree/inverse_parser.svg?style=shield)](https://app.circleci.com/pipelines/github/elena-lyulina/pythonparser?branch=inverse_parser)
 
-This repo contains parsers from **python code** to **xml/json** and vice versa.
-There are parsers for **python2** (see [pythonparser](src/main/python/pythonparser-2.py), source code from [this repo](https://github.com/GumTreeDiff/pythonparser)) and **python3** (see [pythonparser3](src/main/python/pythonparser-3.py), source code from [this repo](https://github.com/Varal7/pythonparser) and [this project](https://eth-sri.github.io/py150)). 
+This repository contains parsers from **python code** to **xml/json** and vice versa.
+There are parsers for **python2** (see [pythonparser](src/main/python/pythonparser-2.py), source code from [this](https://github.com/GumTreeDiff/pythonparser) repository) and **python3** (see [pythonparser3](src/main/python/pythonparser-3.py), source code from [this](https://github.com/Varal7/pythonparser) repository  and [this](https://eth-sri.github.io/py150) project). 
 
+We are going to support Python 3.8 in **python3** parser. 
+Features of Python 3.8 that should affect the parser's code  are: 
+- The "walrus" operator `:=`
+- Positional-only arguments
+- f-strings now support `=`
```

Comment:
Thanks! 
I think that it will look more harmoniously if each item will be href like this:


We are going to support Python 3.8 in **python3** parser:
- [ ] [the "walrus" operator](https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions);
- [ ] [positional-only parameters](https://docs.python.org/3/whatsnew/3.8.html#positional-only-parameters);
- [ ] [f-strings assignment](https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging);

[Here](https://docs.python.org/3/whatsnew/3.8.html) you can read about all new features that Python 3.8 provides.

What do you think?

Reply:
Yes, it is a great idea!